### PR TITLE
ausoceantv: add super basic home and watch pages

### PR DESCRIPTION
**Pull request by trekhopton**
image

image

### CODE REVIEW

File: cmd/ausoceantv/src/logo-holder/logo-holder.css
Comment (scott97) [inline cmd/ausoceantv/src/logo-holder/logo-holder.css:1]: Should this file be deleted given it is empty?

File: cmd/ausoceantv/src/logo-holder/logo-holder.ts
Comment (scott97) [inline cmd/ausoceantv/src/logo-holder/logo-holder.ts:1]: Should this file be deleted given it is empty?

**Review by scott97 — APPROVED** [review]
Left a couple of questions, otherwise LGTM :+1:

**Comment by saxon-milton** [comment]
Closes issue #340

**Pull request merged**